StatisticsInR/2.1_simple_linear_regression.Rmd

```diff
@@ -0,0 +1,1223 @@
+
+</br>
+
+When we look at the uncertainty in this estimate, we see the 95% confidence interval is -0.017 to 0.006. The confidence interval limits are different signs, meaning that 0 is included as a plausible value for the strength of the relationship. Therefore, we cannot conclude that body size has any impact on maximum dive depth in marine mammals. This is supported by the $R^2$ value, which is 0.03, suggesting only 3% of the variation in maximum dive depth is explained by body size. 
+
+</br> 
+
+We should remember here, that when we predicted a dive depth for the fictional whale, it gave us a negative result. Linear models fit a straight line, which can extend beyond the realistic values for some variables. This should be noted and predictions should not be made outside of values that are plausible. This is one reason why predicting outside of the range of your data can be problematic. We can also fit models that are not linear, there is more on those [here](link to GLMs).
+
+</br>
+
+In model checking, there was an interesting pattern shown, of little data at the lowest maximum dive depths and a disproportionate representation of baleen whales. If we plot the data by group and allow ggplot to fit a regression line per group, we can see that there seems to be a different pattern for different taxonomic groups. This would make sense based on physiology and might explain why the variance assumption was not met well AND why the estimated relationship was so weak. 
+
+```{r, echo=T, message=F}
+ggplot(DiveData, aes(x=BodySizeKG, y=MaxDepth, color=Group))+
+  geom_point()+
+  facet_wrap(.~Group)+
+  geom_smooth(method='lm')+
+  theme_minimal()
+```
+*Figure 10: Plot of the relationship between dive depth and body size faceted by taxonomic group.*
+
+</br>
+
+While the baleen whales show no relationship between dive depth and body size, both the pinnipeds (seals and sea lions) and the toothed whales (orca, dolphins, porpoises) show quite strong positive relationships between dive depth and body size. By including all groups in one analysis, we might have been masking the true effects. 
+
+It might be more appropriate to consider an effect of Group as well as body size. We can do this using **Linear models for categorical explanatory variables**/**ANOVA**, check out these pages to continue the analysis of this data.
+
+# What's next? {.facta}
+
+* **Linear models for categorical explanatory variables**/**ANOVA** for analyses when your explanatory variable is not numeric.
+* **Multiple regression** for analyses with more than one numeric explanatory variable.
+* **Generalised linear models** for analyses when your response variable is not normally distributed.
+
```

Comment:
Also add a contributor box. The next version of the css file should have this as an option. I'll also add it to the template file to show how to do this.